vendor/google_riscv-dv/yaml/simulator.yaml

```diff
@@ -57,7 +57,6 @@
       - "vlog -64
         +incdir+<setting>
         +incdir+<user_extension>
-        -access=rwc
```

Comment:
Ah, we can't patch this directly here. I understand that you've already pushed a PR for riscv-dv so the right way to do this to remove this line of this patch, get the riscv-dv change merged and then do a vendoring commit to pull that into Ibex (I'll happily do that last bit if you like: there's a bit of gymnastics involved).

Reply:
Sorry, I was wrong: you'd reported the issue for riscv-dv, but not pushed a PR. I've just pushed [a PR](https://github.com/google/riscv-dv/pull/782) there and will vendor in the fix when it lands.